### ISSUE TITLE
Create Grafana Labs end-user study

### ISSUE THREAD

**Issue by tomkerkhove** [kedacore/governance#79]
See https://grafana.com/blog/2022/10/20/how-to-autoscale-grafana-loki-queries-using-keda/?mdm=social
See https://github.com/kedacore/keda-docs/pull/961#issuecomment-1287115148

**Comment by tomkerkhove** [comment]
@RichiH @bboreham I was wondering if you are still interested in doing this?

**Comment by salvacorts** [comment]
@tomkerkhove IIRC, I send you an email on Feb 8 regarding this. Happy to contribute!
I've answered back to the email chain so you have it back in your inbox. Thank you!

**Comment by tomkerkhove** [comment]
A-ha, there's a bit of confusion - The email thread was for end-user review by CNCF TOC for our Graduation proposal (did they not reach out to you yet?).

However, this thread is to do an end-user study similar to https://keda.sh/blog/2022-03-10-how-zapier-uses-keda/ which we can also check to publish on CNCF blog.

**Comment by tomkerkhove** [comment]
Let me follow up with TOC - Would be a pity they do not interview you.

Wonderful - Are you OK if I reach out with a meeting invite to you next week?

**Comment by salvacorts** [comment]
Absolutely! Looking forward to it. I think @dannykopping should join as well as we have both worked together with Keda.
